### ISSUE TITLE
Prompt template library

### ISSUE THREAD

**Issue by licesonw** [aws-samples/amazon-bedrock-client-for-mac#16]
**Is your feature request related to a problem? Please describe.**
I find myself very often reusing specific prompts to automate tasks during the day (e.g. summarize meeting notes).

**Describe the solution you'd like**
It would be great to have a very lightweight "library" within the app to store those prompts and have them all in the same place. With this in place, I can just quickly insert a prompt template for a specific task i want to automate and get going, without needing to rewrite the prompt or go to another note-taking app where i have saved prompt templates. It could just be a very simple store of txt documents, even without the need for formatting or markdown.

**Describe alternatives you've considered**
A separate note-taking app (like Obsidian) where I store prompt templates, which adds overhead to the workflow.

**Comment by XenMe** [comment]
also would like to have a capability to change prompt, since the default response is too wordy and verbose, and it also impact the speed. to have a feature customize the prompt would more suitable for personalized tasks.

**Comment by didhd** [comment]
This feature has been implemented in the latest version. Thanks for the feedback and patience!
